src/main/java/edu/harvard/iq/dataverse/arquillianglassfishconfig/ArquillianGlassfishConfigurationParser.java

```diff
@@ -0,0 +1,95 @@
+package edu.harvard.iq.dataverse.arquillianglassfishconfig;
```

Comment:
Correct, I thought that in test package we are only holding tests.

Reply:
If some class is used only in tests then there is no problem that we have that places where test classes are. This way is even better because we don't clutter final jar with unnecessary classes.